pkg/cmd/render/render.go

```diff
@@ -245,17 +253,26 @@ func newTemplateData(opts *renderOpts) (*TemplateData, error) {
 	// assume that this is >4.2
 	templateData.Platform = string(infra.Status.PlatformStatus.Type)
 
+	if err := templateData.setBootstrapStrategy(installConfig, opts.delayedHABootstrapScalingStrategyMarker); err != nil {
+		return nil, err
+	}
+
 	if err := templateData.setComputedEnvVars(templateData.Platform); err != nil {
 		return nil, err
 	}
 
-	// Use a marker file to configure the bootstrap scaling strategy.
-	if _, err := os.Stat(opts.delayedHABootstrapScalingStrategyMarker); err == nil {
-		if templateData.NamespaceAnnotations == nil {
-			templateData.NamespaceAnnotations = map[string]string{}
+	// If bootstrap scaling strategy is delayed HA append annotation signal
+	if templateData.BootstrapScalingStrategy == ceohelpers.BootstrapInPlaceStrategy {
```

Comment:
Maybe maybe merge the conditional blocks for setting the annotation and the config map data since they seem to key on the same check?

Reply:
ugh boy :man_facepalming:

Reply:
Thas actually a typo should be `DelayedHAScalingStrategy`